src/tests/ftest/datamover/negative.py

```diff
@@ -214,7 +207,7 @@ def test_dm_negative_error_check_dcp(self):
         dst_path = join(self.posix_local_test_paths[0], "d" * 300)
         self.run_datamover(
             self.test_id + " (filename is too long)",
-            "DAOS_UUID", "/", pool1, cont1,
-            "POSIX", dst_path,
+            src_path=format_path(pool1, cont1)
```

Comment:
(pylint-syntax-error) invalid syntax. Perhaps you forgot a comma? (<unknown>, line 210)